### ISSUE TITLE
Release 2.0.1 not published on NPM public repo

### ISSUE THREAD

**Issue by dabder** [steveathon/bootstrap-wysiwyg#107]
Hi, would you publish the latest version on NPM repo ?

Thanks

**Comment by codewithtyler** [comment]
@dabder Did you mean 2.0.1?

**Comment by codewithtyler** [comment]
@steveathon before releasing on NPM, should the bower.json version number be updated to match the project.json?

**Comment by dabder** [comment]
Yes I guess this is 2.0.1. NPMJS repo indicates 2.0.2 is the latest release (left top hand side) : https://www.npmjs.com/package/bootstrap-wysiwyg

But the latest version published in NPM is 1.0.4 (middle right hand side). npm install command retrieves 1.0.4 version.

About Bower, I don't think NPM minds about this.

**Comment by codewithtyler** [comment]
@dabder looking at the link you provided it looks like that NPM module is managed by someone other than @steveathon. We'd need to contact that person and get them to create a new release. Ideally, it'd be better if we could get them to add @steveathon to the contributors list so he can push a new release himself in the future.

> About Bower, I don't think NPM minds about this.

I'm sure it doesn't, I was just thinking of keeping it consistent to avoid confusing any users.

**Comment by dabder** [comment]
Then I suggest to update bower file to avoid issues/confusions.

I've just contacted the "NPM publisher"

**Comment by artf** [comment]
Hi guys,
I'm the "illegitimate owner" of this repo on NPM and I got the email from @dabder
I'd be happy to move all the permissions to @steveathon but can't find his npm's account

**Comment by codewithtyler** [comment]
I'm not sure if he has one. I'll talk with him about creating one.

**Comment by steveathon** [comment]
@artf thanks Artur, yes I created one as @steveathon - happy if you transfer it to me, but keep edit/update permissions for yourself too.

**Comment by artf** [comment]
@steveathon done :)

**Comment by dabder** [comment]
Thank you guys. Can someone publish ?

**Comment by dabder** [comment]
Morning, can you please publish into the NPM repository. I've just done "NPM install", I still got the old version.

I'm under the pressure to release my app. I would be obliged to use another one if this is not published very soon

Thank you guys

**Comment by artf** [comment]
@dabder done

**Comment by spreadred** [comment]
@RandomlyKnighted Looks like this has been done: https://www.npmjs.com/package/bootstrap-wysiwyg -  This issue can be safely closed.

**Comment by codewithtyler** [comment]
@kaptainkommie sorry, but I don't have the ability to close issues. @steveathon can you close this please?
